GVFS/GVFS.Virtualization/FileSystemCallbacks.cs

```diff
@@ -39,6 +39,7 @@ public class FileSystemCallbacks : IDisposable, IHeartBeatMetadataProvider
         private FileSystemVirtualizer fileSystemVirtualizer;
         private FileProperties logsHeadFileProperties;
         private Thread postFetchJobThread;
+        private GitProcess backgroundGitProcess;

```

Comment:
The word "background" is heavily overloaded in this codebase, so let's find something different. How about `postFetchGitProcess`?